README.md

```diff
@@ -15,30 +15,47 @@ which enables fast read and write operations. It is compatible with a variety
 of platforms and has interfaces for the Fortran, Python, OCaml and Rust
 programming languages.
 
+Below is the list of codes that adopted TREXIO for reading and/or writing single- and/or multi-reference wave functions:
+
+| Software                                                                       | Single-reference | Multi-reference |
+| ------------------------------------------------------------------------------ | ---------------- | --------------- |
+| [Quantum Package](https://github.com/QuantumPackage/qp2)                       | Write/Read       | Write/Read      |
+| [FHI-aims](https://fhi-aims.org/)                                              | Write/Read       | ??              |
+| [PySCF](https://github.com/pyscf/pyscf)                                        | Write/Read       | Write/Read      |
+| [CP2K](https://github.com/cp2k/cp2k)                                           | Write            | ---             |
+| [CHAMP](https://github.com/filippi-claudia/champ)                              | Read             | Read            |
+| [GammCor](https://github.com/pernalk/GAMMCOR)                                  | Read             | Read            |
+| [TurboRVB](https://github.com/sissaschool/turborvb)                            | Read             | ---             |
+| [ipie](https://github.com/JoonhoLee-Group/ipie)                                | Read             | ??              |
+| [Spicy](https://gitlab.com/theoretical-chemistry-jena/quantum-chemistry/Spicy) | Read             | ---             |
+| [QMC=Chem](https://github.com/TREX-CoE/qmcchem2)                               | Read             | Read            |
+| [QMCkl](https://github.com/TREX-CoE/qmckl)                                     | Read             | ---             |
+
```

Comment:
- QP Read/Write multi-ref + Integrals
- FHI-aims Write single-ref + Integrals
- Champ Read multi-ref
- GammCor Read multi-ref + Integrals
- TurboRVB Read single-ref
- ipie Read multi-ref + Integrals
- QMC=Chem read single-ref
- QMCkl read single-ref

Reply:
Thanks! I will update the README